alpakaConfig.cmake

```diff
@@ -366,8 +366,17 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
 
                 SET(CUDA_HOST_COMPILER "${CMAKE_CXX_COMPILER}")
 
-                if(CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
-                    LIST(APPEND CUDA_NVCC_FLAGS "-g" "-G")
+                IF(CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
+                    LIST(APPEND CUDA_NVCC_FLAGS "-g")
+                    # https://github.com/ComputationalRadiationPhysics/alpaka/issues/428
+                    IF(CMAKE_COMPILER_IS_GNUCXX AND
+                       CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0 AND
+                       CUDA_VERSION VERSION_EQUAL 8.0)
```

Comment:
May we need to expand this to CUDA 9+?

Reply:
@psychocoderHPC did not see issues with CUDA 9.0 as he just told me

Reply:
I tested it with CUDA 9. CUDA 9 is not effected.